client/state/posts/test/reducer.js

```diff
@@ -114,7 +147,7 @@ describe( 'reducer', () => {
 				}
 			} );
 		} );
-		it( 'never persists state because this is not implemented', () => {
+		it( 'never persists state', () => {
```

Comment:
I'm going to take a bit of time to play with memoized selectors: http://redux.js.org/docs/recipes/ComputingDerivedData.html